Validations.js

```diff
@@ -0,0 +1,35 @@
+function nameRequired(options) {
+  return _.isString(options.name);
+}
+
+function instrumentRequired(options) {
+  return _.isString(options.guitar) || _.isString(options.drums);
+}
+
+function getNameMessage(options) {
+  return nameRequired(options)
+    ? ""
+    : "Name cannot be empty ";
+}
+
+function getInstrumentMessage(options) {
+  return instrumentRequired(options)
+    ? ""
+    : "A true Rockstar must at least have an instrument";
+}
+
+function getMessage(options) {
+  return getNameMessage(options).concat(getInstrumentMessage(options));
+}
+
+function isValid(options) {
+  return nameRequired(options) && instrumentRequired(options);
+}
+
+function Validate(options) {
+  return {
+    isValid: isValid(options),
+    message: getMessage(options)
```

Comment:
When calling getMessage you are essentially calling the validation method (be it either nameRequired, instrumentRequired or any other method) twice. Try to come up with a design which runs the validation rules only once.